src/pretix/base/settings.py

```diff
@@ -127,11 +127,11 @@ def unserialize(cls, s):
         'serializer_class': serializers.BooleanField,
         'form_kwargs': dict(
             label=_("Ask for email addresses per ticket"),
-            help_text=_("Normally, pretix asks for one email address per order and the order confirmation will be sent "
+            help_text=_("Normally, eventyay asks for one email address per order and the order confirmation will be sent "
                         "only to that email address. If you enable this option, the system will additionally ask for "
                         "individual email addresses for every admission ticket. This might be useful if you want to "
                         "obtain individual addresses for every attendee even in case of group orders. However, "
-                        "pretix will send the order confirmation by default only to the one primary email address, not to "
+                        "eventyay will send the order confirmation by default only to the one primary email address, not to "
```

Comment:
eventyay will send the order confirmation by default only to the primary email address, not to